src/main/java/gregtech/integration/groovy/VirtualizedRecipeMap.java

```diff
@@ -21,20 +22,20 @@ public class VirtualizedRecipeMap extends VirtualizedRegistry<Recipe> {
     private final RecipeMap<?> recipeMap;
 
     public VirtualizedRecipeMap(RecipeMap<?> recipeMap) {
-        super(false, generateAliases(recipeMap.unlocalizedName));
+        super(generateAliases(recipeMap.unlocalizedName));
```

Comment:
Use `Alias.generateOf(recipeMap.unlocalizedName, CaseFormat.LOWER_UNDERSCORE)` and remove the `generateAlias`method